docs/pages/repo/docs/reference/command-line-reference.mdx

```diff
@@ -178,6 +178,55 @@ Task details include:
 - `dependencies`: Tasks that must run before this task
 - `dependents`: Tasks that must be run after this task
 
+#### `--experimental-env-mode`
+
+`type: string`
+
+<Callout type="info">
+  **Warning**: This in experimental flag, so its name and behavior can change.
```

Comment:
Typo. 😄 "This is an experimental"

Reply:
good catch. you can make suggestions from the UI btw. makes it easy to merge in typo changes :)